tests/chainer_tests/functions_tests/activation_tests/test_leaky_relu.py

```diff
@@ -28,31 +28,27 @@ def setUp(self):
         self.ggx = numpy.random.uniform(-1, 1, self.shape).astype(self.dtype)
         self.slope = random.random()
         self.check_forward_options = {}
-        self.check_backward_options = {'dtype': numpy.float64}
+        self.check_backward_options = {}
+        self.check_double_backward_options = {}
         if self.dtype == numpy.float16:
             self.check_forward_options = {'atol': 1e-4, 'rtol': 1e-3}
-            self.check_backward_options = {
-                'dtype': numpy.float64, 'atol': 5e-3, 'rtol': 5e-2}
+            self.check_backward_options = {'atol': 1e-4, 'rtol': 1e-3}
+            self.check_double_backward_options = {'atol': 5e-4, 'rtol': 5e-3}
```

Comment:
In my environment, this configuration is too strict (both backward and double-backward).
Can you relax it?